docs/openid.md

```diff
@@ -336,11 +336,12 @@ oidc_providers:
        issuer: "https://accounts.google.com/"
        client_id: "your-client-id" # TO BE FILLED
        client_secret: "your-client-secret" # TO BE FILLED
-       scopes: ["openid", "profile"]
+       scopes: ["openid", "profile", "email"] # email is optional, read below
```

Comment:
I'm guessing this corresponds to the three scopes listed here?

https://developers.google.com/identity/protocols/oauth2/scopes#oauth2

Reply:
Sanity check: have you used this config yourself and confirmed that it works? (On matrix.org we are using the scope name `https://www.googleapis.com/auth/userinfo.email`.)